src/main/java/com/studioedge/focus_to_levelup_server/domain/focus/service/FocusService.java

```diff
@@ -0,0 +1,79 @@
+package com.studioedge.focus_to_levelup_server.domain.focus.service;
+
+import com.studioedge.focus_to_levelup_server.domain.character.entity.MemberCharacter;
+import com.studioedge.focus_to_levelup_server.domain.character.exception.CharacterDefaultNotFoundException;
+import com.studioedge.focus_to_levelup_server.domain.character.repository.MemberCharacterRepository;
+import com.studioedge.focus_to_levelup_server.domain.focus.dao.DailyGoalRepository;
+import com.studioedge.focus_to_levelup_server.domain.focus.dao.SubjectRepository;
+import com.studioedge.focus_to_levelup_server.domain.focus.dto.request.SaveFocusRequest;
+import com.studioedge.focus_to_levelup_server.domain.focus.dto.response.FocusModeAnimationResponse;
+import com.studioedge.focus_to_levelup_server.domain.focus.entity.DailyGoal;
+import com.studioedge.focus_to_levelup_server.domain.focus.entity.Subject;
+import com.studioedge.focus_to_levelup_server.domain.focus.exception.DailyGoalNotFoundException;
+import com.studioedge.focus_to_levelup_server.domain.focus.exception.SubjectNotFoundException;
+import com.studioedge.focus_to_levelup_server.domain.focus.exception.SubjectUnAuthorizedException;
+import com.studioedge.focus_to_levelup_server.domain.member.dao.MemberRepository;
+import com.studioedge.focus_to_levelup_server.domain.member.entity.Member;
+import com.studioedge.focus_to_levelup_server.domain.member.exception.MemberNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDate;
+
+@Service
+@RequiredArgsConstructor
+public class FocusService {
+    private final MemberRepository memberRepository;
+    private final SubjectRepository subjectStatRepository;
```

Comment:
```suggestion
    private final SubjectRepository subjectRepository;
```
오타 수정 필요해 보입니다. 아래에서 이를 사용하는 부분들도 수정 필요합니다.